core/src/test/scala/org/broadinstitute/dsde/rawls/snapshot/SnapshotServiceSpec.scala

```diff
@@ -135,7 +147,7 @@ class SnapshotServiceSpec extends AnyWordSpecLike with Matchers with MockitoSuga
       val mockWorkspaceManagerDAO = defaultMockWorkspaceManagerDao()
       when(
         mockWorkspaceManagerDAO.getWorkspace(any[UUID], any[RawlsRequestContext])
-      ).thenAnswer(_ => new ApiException(404, "Workspace does not exist"))
+      ).thenAnswer(_ => throw new ApiException(404, "Workspace does not exist"))
```

Comment:
I bumped into some type problems here that precipitated this change, I don't know exactly why they only showed up after I started moving things around though.  But specifically, my interpretation of the compiler error was that it was treating the `ApiException` as the return value rather than as a thrown value, and the compiler couldn't cast it to the expected return type.